### PR TITLE
Feat/aggregations (reactivesearch React and Vue)

### PR DESCRIPTION
**Pull request by ShahAnuj2610**
aggregationField for `react` and `vue`

### CODE REVIEW

File: content/docs/reactivesearch/vue/search/DataSearch.md [hunk 1]
```diff
@@ -61,6 +61,26 @@ Example uses:
...
+    	slot-scope="{
+            ...
+            aggregationData
+        }"
+    >
+    	...
+    </template>
+    ```
+
+    > If you are using an app with elastic search version less than 6, then you have to handle error manually using **renderError** slot.
```
Comment (bietkul) [inline content/docs/reactivesearch/vue/search/DataSearch.md]: This sentence seems like that the user will have an error if using the es6 apps disregarding of they are using `aggregationField` or not.
Just mention that this prop doesn't work with es6 apps (no support of composite aggregation), it'll throw the error which can be handled by using `onError` or `renderError`.

File: content/docs/reactivesearch/vue/search/DataSearch.md [hunk 2]
```diff
@@ -61,6 +61,26 @@ Example uses:
     unique identifier of the component, can be referenced in other components' `react` prop.
 -   **dataField** `String or Array`
     database field(s) to be queried against. Accepts an Array in addition to String, useful for applying search across multiple fields.
+-   **aggregationField** `String` [optional]
+    One of the most important use-cases this enables is showing `DISTINCT` results (useful when you are dealing with sessions, events and logs type data). It utilizes `composite aggregations` which are newly introduced in ES v6 and offer vast performance benefits over a traditional terms aggregation.
+    You can read more about it over [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-composite-aggregation.html). You can access `aggregationData` using `renderAllSuggestions` slot as shown:
+
+    ```html
+    <template
+    	slot="renderAllSuggestions"
```
Comment (bietkul) [inline content/docs/reactivesearch/vue/search/DataSearch.md:70]: @ShahAnuj2610 You also need to add the `aggregationField` in the docs of `renderAllSuggestions`.

File: content/docs/reactivesearch/vue/result/ReactiveList.md [hunk 3]
```diff
@@ -52,6 +52,26 @@ Results.." sortBy="desc" :stream="true" :pagination="false" :pages="5" :size="10
...
+    	slot-scope="{
+            ...
+            aggregationData
+        }"
+    >
+    	...
+    </template>
+    ```
+
+    > If you are using an app with elastic search version less than 6, then you have to handle error manually using **renderError** slot.
```
Comment (bietkul) [inline content/docs/reactivesearch/vue/result/ReactiveList.md]: Read the comment for `data-search`.

File: content/docs/reactivesearch/v3/search/datasearch.md [hunk 4]
```diff
@@ -63,6 +63,21 @@ Example uses:
...
+    You can read more about it over [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-composite-aggregation.html). You can access `aggregationData` using render prop as shown:
+
+    ```javascript
+    <DataSearch
+        aggregationField="original_title.keyword"
+        render={({aggregationData}) => {...}}
+    />
+    ```
+
+    > If you are using an app with elastic search version less than 6, then you have to handle error manually using **renderError** prop.
```
Comment (bietkul) [inline content/docs/reactivesearch/v3/search/datasearch.md]: Check comments for data search for vue.

File: content/docs/reactivesearch/v3/search/categorysearch.md [hunk 5]
```diff
@@ -68,6 +68,21 @@ Example uses:
     unique identifier of the component, can be referenced in other components' `react` prop.
 -   **dataField** `String or Array`
     database field(s) to be queried against. Accepts an Array in addition to String, useful for applying search across multiple fields.
+-   **aggregationField** `String` [optional]
```
Comment (bietkul) [inline content/docs/reactivesearch/v3/search/categorysearch.md:71]: Update `render` prop docs.

File: content/docs/reactivesearch/v3/result/reactivelist.md [hunk 6]
```diff
@@ -66,6 +66,21 @@ Example uses:
...
+    You can read more about it over [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-bucket-composite-aggregation.html). You can access `aggregationData` using render prop as shown:
+
+    ```javascript
+    <ReactiveList
+        aggregationField="original_title.keyword"
+        render={({aggregationData}) => {...}}
+    />
+    ```
+
+    > If you are using an app with elastic search version less than 6, then you have to handle error manually using **renderError** prop.
```
Comment (bietkul) [inline content/docs/reactivesearch/v3/result/reactivelist.md]: Update this everywhere.

**Review by bietkul — CHANGES_REQUESTED** [review]

**Comment by ShahAnuj2610** [comment]
@bietkul have made the changes

**Review by bietkul — APPROVED** [review]

**Pull request merged**